onchain/rollups/contracts/dapp/CartesiDApp.sol

```diff
@@ -117,14 +117,14 @@ contract CartesiDApp is
         bytes32 epochHash;
         uint256 firstInputIndex;
         uint256 lastInputIndex;
-        uint256 inboxInputIndex;
+        uint256 inputIndex;
 
         // query the current consensus for the desired claim
         (epochHash, firstInputIndex, lastInputIndex) = getClaim(_proof.context);
 
         // validate the epoch input index and calculate the inbox input index
```

Comment:
Here, remove the word "inbox".
This also applies to the documentation of `validateInputIndexRange`.